src/Microsoft.ML.Core/Data/ITransformModel.cs

```diff
@@ -25,6 +25,10 @@ public interface ITransformModel
         /// </summary>
         ISchema InputSchema { get; }
 
+        /// <summary>
+        /// Schema of the transform model.
+        /// </summary>
+        IDataView Schema { get; }
```

Comment:
I think the name `Schema` here seems bit misleading. The type here is `IDataView`. This is against `InputSchema` defined above which has type `ISchema`.